src/components/Picker/Picker.scss

```diff
@@ -126,7 +126,7 @@ $stacking-order: (
 .PopoverButtonWrap {
   min-height: 30px;
   position: absolute;
-  min-width: 42rem;
+  max-width: 42rem;
```

Comment:
Hi @Farhan-emgage this is not the desired fix, as it will wrap long names in the picker. Can you please look into this more. Thanks

Reply:
@weegrom actually main issue is previously the popover of picker has no width. but when button appear at that time min-width is applying. because of that the popover is creating issue with width.

Reply:
@Farhan-emgage setting `max-width: 42rem;` is not the appropriate fix. In this case a picker option over 42rem will not display properly or wrap. We set 42rem min width to keep picker options such as [A, B, C, D, F] from displaying in dropdown that's so thin it's difficult to interact with. The dropdown width should be based on it's content, and only limitied by the browser screen width.

Reply:
ok @weegrom will look after it